.github/CODEOWNERS

```
@@ -1 +1 @@
-* @google-github-actions/maintainers
+* @google-github-actions/maintainers @google-github-actions/analyze-code-security-scc-maintainers
```

I'd prefer to require our approval to change CODEOWNERS
```suggestion
* @google-github-actions/maintainers @google-github-actions/analyze-code-security-scc-maintainers

.github/CODEOWNERS @google-github-actions/maintainers
```

or retain approvals over that entire directory:

```suggestion
* @google-github-actions/maintainers @google-github-actions/analyze-code-security-scc-maintainers

.github/ @google-github-actions/maintainers
```